Model/AbstractRequest.php

```diff
@@ -318,16 +348,16 @@ protected function getShippingAddressParams(Quote $quote)
     }
 
     /**
-     * @param Quote $quote
+     * Get Shipping lines parameters
      *
+     * @param Quote $quote
      * @return array
      */
     protected function getShippingLinesParams(Quote $quote)
     {
         $params = [];
 
-        if (
-            ($shipping = $quote->getShippingAddress()) !== null &&
+        if (($shipping = $quote->getShippingAddress()) !== null &&
             ($rate = $shipping->getShippingRatesCollection()->getFirstItem()) !== null
```

Comment:
I think you can drop the null check and it will work the same:
```
if (
  ($shipping = $quote->getShippingAddress()) &&
  ($rate = $shipping->getShippingRatesCollection()->getFirstItem())
)
```